src/App.tsx

```diff
@@ -1,36 +1,32 @@
 import "focus-visible/dist/focus-visible"
 import { FC } from "react"
-import { Box, ChakraProvider, Container } from "@chakra-ui/react"
+import { ChakraProvider } from "@chakra-ui/react"
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom"
 import theme from "./theme"
 import Navbar from "./components/Navbar"
 import SplashPage from "./pages/Splash"
-import Swoosh from "./static/images/Swoosh.svg"
+import useDarkMode from "./hooks/useDarkMode"
 
-const App: FC = () => {
+const App = () => {
+  useDarkMode()
```

Comment:
@r-czajkowski This hook forces the app into darkmode which defaults Boxes to have a black bg and text to have white color, among other welcome changes.. There will be no lightmode on this app, so I think this hook should be ok to force dark mode and save us the headache of updating all the styles to dark.

Reply:
This solution works for me but I think we can force dark mode by setting up [the chakra config](https://chakra-ui.com/docs/styled-system/features/color-mode#updating-the-theme-config) and wrapping the whole app with [`DarkMode` component](https://chakra-ui.com/docs/styled-system/recipes/page-specific-color-mode).

theme
```
import { extendTheme } from '@chakra-ui/react'

// 2. Add your color mode config
const config = {
  initialColorMode: 'dark',
  useSystemColorMode: false,
}

// 3. extend the theme
const theme = extendTheme({ config })

export default theme
```

App.tsx
```
import { DarkMode, GlobalStyle } from '@chakra-ui/react'
<DarkMode>
  <GlobalStyle />
  {/* rest of components */}
</DarkMode>
```